adminSDK/directory/index.html

```diff
@@ -78,7 +78,7 @@
        */
       function gisLoaded() {
         tokenClient = google.accounts.oauth2.initTokenClient({
-          client_id: CLIENT_ID,
+          client_id: 692918908620-rfum1gdqtc0bfkcgk5gd7uonupe1tm4i.apps.googleusercontent.com,
```

Comment:
what even is this PR? are you trying to merge your client_id to the official docs?